src/Sylius/Bundle/ShopBundle/Resources/views/Cart/Summary/_item.html.twig

```diff
@@ -1,16 +1,24 @@
 {% import "@SyliusShop/Common/Macro/money.html.twig" as money %}
 
 {% set product_variant = item.variant %}
-{% set original_price_to_display = sylius_order_item_original_price_to_display(item, cart.channel) %}
+{% set original_price = null %}
+
+{% if item.originalUnitPrice is not null and item.originalUnitPrice > item.unitPrice %}
+    {% set original_price = item.originalUnitPrice %}
+{% elseif item.originalUnitPrice is not null and item.originalUnitPrice > item.discountedUnitPrice %}
+    {% set original_price = item.originalUnitPrice %}
+{% elseif item.originalUnitPrice is null and item.unitPrice > item.discountedUnitPrice %}
+    {% set original_price = item.unitPrice %}
+{% endif %}
```

Comment:
A lot of logic in the twig :dancer: Cannot we simplify it a little bit (1st and 2nd conditions have the same result)? Or maybe even move it to `OrderItem::getOriginalUnitPrice(): int`?

Reply:
Or eventually bring back the removed extension and put the logic there

Reply:
Moved logic to the removed extension class